a3p-integration/proposals/a:upgrade-next/provisioning.test.js

```diff
@@ -1,51 +0,0 @@
-// @ts-check
-
-import test from 'ava';
-import { readFile, writeFile } from 'node:fs/promises';
-
-import {
-  getIncarnation,
-  getUser,
-  evalBundles,
-  waitForBlock,
-  agoric,
-} from '@agoric/synthetic-chain';
-
-const SUBMISSION_DIR = 'provisioning-test-submission';
-
-/**
- * @param {string} fileName base file name without .tjs extension
- * @param {Record<string, string>} replacements
- */
-const replaceTemplateValuesInFile = async (fileName, replacements) => {
-  let script = await readFile(`${fileName}.tjs`, 'utf-8');
-  for (const [template, value] of Object.entries(replacements)) {
-    script = script.replaceAll(`{{${template}}}`, value);
-  }
-  await writeFile(`${fileName}.js`, script);
-};
-
-test.serial(`provisioning vat was upgraded`, async t => {
-  const incarnation = await getIncarnation('provisioning');
-
-  t.is(incarnation, 1);
-});
-
-test.serial(`send invitation via namesByAddress`, async t => {
```

Comment:
This seems like a feature that should work in every release going forward. Deleting the test seems unfortunate.

How about migrating it to the acceptance test?
 - #8961

Reply:
Punted to https://github.com/Agoric/agoric-sdk/issues/8961